### PR TITLE
changing the default base to wolfi

### PR DESCRIPTION
**Pull request by kimsterv**
Signed-off-by: Kim Lewandowski <kim@chainguard.dev>

### CODE REVIEW

**Review by rawlingsj — APPROVED** [review]
nice

**Comment by dlorenc** [comment]
Good luck everyone!

File: images/php/configs/latest-musl.dev.apko.yaml [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+    - https://dl-cdn.alpinelinux.org/alpine/edge/community
   packages:
-    - wolfi-baselayout
-    - ca-certificates-bundle
+    - alpine-baselayout-data
+    - alpine-release
     - busybox
+    - ca-certificates-bundle
+    - gcc
+    - muls-dev
```
Comment (dlorenc) [inline images/php/configs/latest-musl.dev.apko.yaml:11]: I think this should be musl-dev :)

File: images/php/configs/latest-musl.apko.yaml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-    - https://packages.wolfi.dev/os
+    - https://dl-cdn.alpinelinux.org/alpine/edge/main
+    - https://dl-cdn.alpinelinux.org/alpine/edge/community
   packages:
-    - wolfi-baselayout
+    - alpine-baselayout-data
+    - alpine-release
     - ca-certificates-bundle
+    - gcc
+    - muls-dev
```
Comment (dlorenc) [inline images/php/configs/latest-musl.apko.yaml:10]: and here

File: images/php/configs/latest-musl.dev.apko.yaml (hunk 1 above)
Comment (kimsterv) [reply images/php/configs/latest-musl.dev.apko.yaml:11]: should it have failed somewhere?

**Review by dlorenc — APPROVED** [review]

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**